crates/store/src/genesis/config/samples/01-simple.toml

```diff
@@ -0,0 +1,15 @@
+timestamp = 1717344256
+version   = 1
+
+[[fungible_faucet]]
+decimals     = 12
+max_supply   = 1000000
+storage_mode = "public"
+symbol       = "MIDEN"
+# fungible = true # default value
```

Comment:
Same as one of the previous comments: I don't think we need `fungible` flag because we already define this as a fungible faucet.